### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by Ashnaali3255**
**In this branch I did:**

- Create a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. This method should do the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require './app'
+def main
+  app = App.new
+  app.run
```
Comment (TedLivist) [inline main.rb:5]: - [ ] Kindly ensure your app works. There is no method for running your app. As soon as I run `ruby main.rb`, the app crashes. Kindly look carefully at the video you are to emulate and ensure your app works like that.

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Ashnaali3255,

Good job so far!
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ashnaali3255** [comment]
Dear Code Reviewer!! there is a linter error which is this:
**app.rb:150:3: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for handle_choice is too high. [8/7]
  def handle_choice(choice) ...**
  So i can't remove my code because my code can't updated more than this.

**Comment by Felix45** [comment]
> Dear Code Reviewer!! there is a linter error which is this: **app.rb:150:3: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for handle_choice is too high. [8/7] def handle_choice(choice) ...** So i can't remove my code because my code can't updated more than this.

Please check your slack messages, I've inboxed you.

**Review by algerina — COMMENTED** [review]
Hi @Ashnaali3255 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a linter errors.
Meanwhile you can take a look to that l[ink ](https://stackoverflow.com/questions/47336195/cyclomatic-complexity-for-some-method-is-too-high)and try the solution provided.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Ashnaali3255 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- UI working as expected ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**